2.7.0/final.md

```diff
@@ -0,0 +1,958 @@
+  and the subsequent local SDPAs will have to wait until the all-gather completes, and offered a user API ([#142093](https://github.com/pytorch/pytorch/pull/142093)) to select the desired approach. The implementation
+  currently supports three SDPA kernels: `SDPBackend.FLASH_ATTENTION`, `SDPBackend.EFFICIENT_ATTENTION`, and `SDPBackend.CUDNN_ATTENTION` ([#148537](https://github.com/pytorch/pytorch/pull/148537)). We also
+  verified that our Context Parallel implementation is compatible with other parallelisms and `torch.compile`.
+### c10d
+- Implemented ncclCommInitRankScalable (merging #136789) ([#144794](https://github.com/pytorch/pytorch/pull/144794))
+### Distributed Checkpoint (DCP)
+- Cache save plans: to mitigate overhead from planning steps ([#147116](https://github.com/pytorch/pytorch/pull/147116), [#147343](https://github.com/pytorch/pytorch/pull/147343))
+- Build a storage reader/writer to write checkpoints in HF format ([#148089](https://github.com/pytorch/pytorch/pull/148089))
+
+## CUDA
+- RTX50 Blackwell Support codegen ([#145270](https://github.com/pytorch/pytorch/pull/145270))
+- Make `torch.cuda.gds` APIs public ([#147120](https://github.com/pytorch/pytorch/pull/147120))
+
+## MPS
+
+## ROCm
+- CK Memory-Efficient Attention (attention bias support) (#147778)
+- CK Flash Attention Backend (#143695)
+- Enhanced Windows support for PyTorch on ROCm (#148563, #144098)
+- Support for gfx1102 arch (Navi33) in wheel builds (#147761)
+- hipblaslt rowwise f8 gemm (#144432)
+
+## XPU
+- Add AOT Inductor support for Intel GPU ([#140269](https://github.com/pytorch/pytorch/pull/140269), [#140664](https://github.com/pytorch/pytorch/pull/140664), [#149175](https://github.com/pytorch/pytorch/pull/149175))
+- Support `torch.compile` on Windows Platform for XPU ([#147637](https://github.com/pytorch/pytorch/pull/147637), [#144316](https://github.com/pytorch/pytorch/pull/144316))
+- Support SYCL with `torch.utils.cpp_extension` APIs ([#132945](https://github.com/pytorch/pytorch/pull/132945))
+- Enhance Intel GPU performance on PyTorch 2 Export Post Training Quantization ([#136753](https://github.com/pytorch/pytorch/pull/136753), [#135465](https://github.com/pytorch/pytorch/pull/135465),[#135337](https://github.com/pytorch/pytorch/pull/135337), [#135189](https://github.com/pytorch/pytorch/pull/135189))
+- Enable windows Kineto profiler([#148319](https://github.com/pytorch/pytorch/pull/148319))
+- Enable TF32 support for XPU based on oneDNN backend ([#137570](https://github.com/pytorch/pytorch/pull/137570))
+
+## torch.dynamo
+- Support tracing `contextlib.contextmanager` in Dynamo (#136033)
+- `nonstrict_trace` escape hatch to apply non-strict tracing to difficult-to-compile code (#146367)
+- Delayed compile for dynamic shapes (#147983)
+- Support tracing generators (#141055)
+- Whitelist of source files to apply dynamic shapes to (#147979)
+- Support tracing `list` subclasses (#146819)
+
+## torch.inductor
```

Comment:
I think having these subcategories of torch.compile would be more consistent with other categories like distributed. (

Reply:
Yea I was considering that--would you also consider fx and export as under torch.compile?

Reply:
you should do torch.compile, torch.export, and torch.fx as separate categories. Depending on the torch.fx commits, then can go into torch.compile or torch.export (if they were done for the purpose of compile or export)